src/main/java/loanbook/model/loan/Email.java

```diff
@@ -34,6 +34,21 @@ public Email(String email) {
         super(MESSAGE_EMAIL_CONSTRAINTS, Email::isValidEmail, Function.identity(), email);
     }
 
+    /**
+     * Check if user's new {@code Email} is a valid Gmail.
+     */
+    public static boolean isValidGmail(String email) {
```

Comment:
Perhaps we can use some regex here?
Try this: `^[a-z0-9](\.?[a-zA-Z0-9])+@gmail\.com$`

Essentially, it will allow alphanumeric and dots in the username, but it must end in @gmail.com

Reply:
Fixed.